Bottom: 6f5cf1816c5e57921d13b1047cf5423269825419
Top:    6192b3e7a0ac4845b19709be8603490b0c742ea2
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2016-03-02 16:09:37 -0300

Refresh of char-type.patch

---

diff --git a/include/gcc-c-fe.def b/include/gcc-c-fe.def
index 4b32ff4..3f3679b 100644
--- a/include/gcc-c-fe.def
+++ b/include/gcc-c-fe.def
@@ -134,7 +134,7 @@ GCC_METHOD2 (gcc_type, int_type,
 /* Return the 'char' type, a distinct type from both 'signed char' and
    'unsigned char' returned by int_type.  */
 
-GCC_METHOD2 (gcc_type, char_type)
+GCC_METHOD0 (gcc_type, char_type)
 
 /* Return a floating point type with the given properties.  */
 
diff --git a/include/gcc-cp-fe.def b/include/gcc-cp-fe.def
index 74af292..bbd6b8f 100644
--- a/include/gcc-cp-fe.def
+++ b/include/gcc-cp-fe.def
@@ -513,7 +513,7 @@ GCC_METHOD2 (gcc_type, int_type,
 /* Return the 'char' type, a distinct type from both 'signed char' and
    'unsigned char' returned by int_type.  */
 
-GCC_METHOD2 (gcc_type, char_type)
+GCC_METHOD0 (gcc_type, char_type)
 
 /* Return a floating point type with the given properties.  */
